config.go

```diff
@@ -889,6 +891,18 @@ func loadConfig() (*config, []string, error) {
 			oldTBConfigFile, configFilePath)
 	}
 
+	// Warn if user still is still using --ticketaddress
+	if cfg.TicketAddress.Address != nil {
+		log.Warnf("--ticketaddress has been DEPRECATED.  Use " +
+			"--ticketbuyer.votingaddress instead")
+		if cfg.TBOpts.VotingAddress.Address == nil {
+			cfg.TBOpts.VotingAddress = cfg.TicketAddress
```

Comment:
I really don't like reassigning the flag.  Can you create a local var for the address instead and use that below?